src/backend/base/langflow/components/nvidia/nvidia_ingest.py

```
@@ -162,6 +178,17 @@
             self.log(err_msg)
             raise ValueError(err_msg)
 
+        # Check if all files are PDFs when high resolution mode is enabled
+        if self.high_resolution:
+            for file in file_list:
+                try:
+                    with open(file.path, "rb") as f:
```

Ruff (0.11.9)

`open()` should be replaced by `Path.open()`

(PTH123)